src/main/java/fr/esgi/al5_2/Tayarim/services/AdministrateurService.java

```diff
@@ -0,0 +1,171 @@
+package fr.esgi.al5_2.Tayarim.services;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.*;
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
+import fr.esgi.al5_2.Tayarim.exceptions.*;
+import fr.esgi.al5_2.Tayarim.mappers.AdministrateurMapper;
+import fr.esgi.al5_2.Tayarim.mappers.ProprietaireMapper;
+import fr.esgi.al5_2.Tayarim.repositories.AdministrateurRepository;
+import fr.esgi.al5_2.Tayarim.repositories.ProprietaireRepository;
+import lombok.NonNull;
+import org.mindrot.jbcrypt.BCrypt;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Transactional(readOnly = true)
+public class AdministrateurService {
+
+  private final AdministrateurRepository administrateurRepository;
+
+  public AdministrateurService(AdministrateurRepository administrateurRepository) {
+    this.administrateurRepository = administrateurRepository;
+  }
+
+  @Transactional
+  public AdministrateurDTO creerAdministrateur(
+      @NonNull AdministrateurCreationDTO administrateurCreationDTO) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'administrateurCreationDTO' must contain no more than '1' consecutive capital letters.